### ISSUE TITLE
Add faculty filter

### ISSUE THREAD

**Issue by jirutka** [cvut/sirius#108]
FEL guys would like to filter events that are relevant for them, i.e. courses which are taught by their teachers or attended by their students. This is a quite complicated issue, but Umapi already did the hard work.

**Comment by jnv** [comment]
Like, what? `/faculties/[id]/events`?

**Comment by jirutka** [comment]
I think that query parameter would be better.

**Comment by jnv** [comment]
I don't think so. Furthermore, how do we infer association? By having m:n from people to faculties?

**Comment by jnv** [comment]
Or, wait, actually for events you'd have to precalculate it. Dammit, those criteria don't make much sense to me.

**Comment by jirutka** [comment]
Calm down, it’s much difficult issue than you think… :smirk_cat: Therefore it has low priority and it’s not a time for it now.
